content/guides/basic-setup/index.md

```diff
@@ -13,48 +13,55 @@ If you require further help, please visit our forums or ask in our Discord's hel
 
 ## How to dump your PS2 BIOS
 
-Dumping your PS2 BIOS requires a mod to your PS2 to allow arbitrary code execution.
+Dumping your PS2 BIOS is conceptually a two-step process:
+1. "Hack" your PS2 to allow arbitrary code execution
+2. Now you can run a "BIOS dumper" utility program on your PS2 that reads its BIOS and writes it to a USB drive.
 
-**Popular options:**
+There is a generally useful program, uLaunchELF, that lets you browse memory cards, DVDs, and USB drives connected to a PS and run programs from them. So in most of the approaches below, you "hack" your PS2 so that it runs uLaunchELF, and uLaunchELF in turn it lets you run the BIOS dumper utility from a USB drive.
```

Comment:
`uLaunchELF, and uLaunchELF in turn it lets you run the BIOS dumper utility from a USB drive` -> 
`uLaunchELF and use it to run the BIOS dumper utility from a USB drive.`

Reply:
Thanks for looking at my edits. I'm not expert in pull request iteration, do you want me to incorporate updates and propose a new pull request, or will you do it?

Reply:
Ah ok, if you aren't familiar with git i can make the edits as per the discussion below.  It'll be a day or so before i get around to it though.